uktrade/data-hub-frontend
# Events collection page (except filters)

## Description of change

Sets up the events collection list page in react - including:

- Render each item in the list with badges and metadata
- Sort by options (note that the default sort by will be applied when the filters are added)
- Add event button

## Test instructions

Go to `events/react`

The collection list should appear, with sort by options and a button to add an event.

## Screenshots

![Screenshot from 2021-06-21 17-07-08](https://user-images.githubusercontent.com/1234577/122793620-53b0b280-d2b3-11eb-83df-01c839ab2ecc.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
